src/main/java/com/denizenscript/depenizen/bukkit/objects/superiorskyblock/SuperiorSkyblockIslandTag.java

```diff
@@ -0,0 +1,386 @@
+            return new ElementTag(object.getIsland().getIslandSize());
+        });
+
+        // <--[tag]
+        // @attribute <SuperiorSkyblockIslandTag.total_rating>
+        // @returns ElementTag(Decimal)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Returns the current average rating of the specified island.
+        // -->
+        tagProcessor.registerTag(ElementTag.class, "total_rating", (attribute, object) -> {
+            return new ElementTag(object.getIsland().getTotalRating());
+        });
+
+        // <--[mechanism]
+        // @object SuperiorSkyblockIslandTag
+        // @name balance
+        // @input ElementTag(Decimal)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Changes an island's bank balance.
+        // @tags
+        // <SuperiorSkyblockIslandTag.balance>
+        // -->
+        tagProcessor.registerMechanism("balance", false, ElementTag.class, (object, mechanism, value) -> {
+            if (mechanism.requireDouble()) {
+                object.getIsland().getIslandBank().setBalance(mechanism.getValue().asBigDecimal());
+            }
+        });
+
+        // <--[mechanism]
+        // @object SuperiorSkyblockIslandTag
+        // @name description
+        // @input ElementTag
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Changes an island's description. Leave blank to remove.
+        // @tags
+        // <SuperiorSkyblockIslandTag.description>
+        // -->
+        tagProcessor.registerMechanism("description", false, ElementTag.class, (object, mechanism, value) -> {
+            object.getIsland().setDescription(mechanism.getValue().asString());
+        });
+
+        // <--[mechanism]
+        // @object SuperiorSkyblockIslandTag
+        // @name locked
+        // @input ElementTag(Boolean)
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Changes whether an island is locked to visitors.
+        // @tags
+        // <SuperiorSkyblockIslandTag.locked>
+        // -->
+        tagProcessor.registerMechanism("locked", false, ElementTag.class, (object, mechanism, value) -> {
+            if (mechanism.requireBoolean()) {
+                object.getIsland().setLocked(mechanism.getValue().asBoolean());
+            }
+        });
+
+        // <--[mechanism]
+        // @object SuperiorSkyblockIslandTag
+        // @name name
+        // @input ElementTag
+        // @plugin Depenizen, SuperiorSkyblock
+        // @description
+        // Changes an island's name.
+        // @tags
+        // <SuperiorSkyblockIslandTag.name>
+        // -->
+        tagProcessor.registerMechanism("name", false, ElementTag.class, (object, mechanism, value) -> {
+            if (!mechanism.getValue().asString().isEmpty() && SuperiorSkyblockAPI.getIsland(mechanism.getValue().asString()) == null) {
```

Comment:
needs an else echoError. Also document what the expected limitations are in the meta